cmsplugin_carousel_ai/models.py

```diff
@@ -52,11 +70,21 @@ class Slide(models.Model):
     image = FilerImageField(
         verbose_name=_("slide image"),
     )
-    caption = models.CharField(
-        verbose_name=_("slide caption"),
+    header = models.CharField(
+        verbose_name=_("slide header"),
+        max_length=160,
+        blank=True,
+    )
+    subheader = models.CharField(
+        verbose_name=_("slide subheader"),
         max_length=160,
         blank=True,
     )
+    content = models.CharField(
+        verbose_name=_("slide content"),
+        max_length=160,
+        blank=True,
+    ) 
```

Comment:
`header`, `subheader` and `content` all seem to be new fields unrelated to the topic of this PR.